packages/api/src/routes/medical/index.ts

```diff
@@ -16,8 +16,10 @@ routes.use("/facility", facilityRoot);
 routes.use("/facility/:id", facilityAuthorization("params"), facility);
 
 routes.use("/patient", patientRoot);
+// patient routes are also used in EHR Integrations
```

Comment:
nit: "...also used in EHR Integrations routes"? Just to add a bit more context and help the reader to find the places being referenced

Reply:
Done.